pkg/configuration/configuration_test.go

```diff
@@ -213,6 +214,21 @@ func TestConfiguration_WithEnvironmentVariable_ReturnsEnvironmentVariableValue(t
 	assert.Equal(expected, actual)
 }
 
+func TestSubConfiguration_WithEnvironmentVariable_ReturnsEnvironmentVariableValue(t *testing.T) {
```

Comment:
I feel it wouldn't hurt to add a test for the subconfiguration, it seems this is the only one, I'd like to have one which checks that an entire sub-tree is correct.

Reply:
Added